CHANGELOG.md

```diff
@@ -2,42 +2,36 @@
 
 This file is used to list changes made in each version of the yum-centos cookbook.
 
+## Unreleased
+
+- Remove CentOS 6 support
+- Update vault repo versions to latest
+- MDL fixes
+- Cleanup metadata.rb formatting
+
 ## 4.0.2 (2020-07-21)
 
-### Fixed
 - Fix Chef 12 compatibility (resolves #42)
-
-### Added
 - Kitchen suite for testing against Chef 12
 
 ## 4.0.1 (2020-07-20)
 
-### Added
```

Comment:
why are all the headings removed?

Reply:
Figured it looked better without them in the end.

Reply:
I wouldn't add these new, but we should keep the existing ones since they break down the changes into types nicely